doc/building_sdk.md

```diff
@@ -0,0 +1,30 @@
+#### Building SDK
```

Comment:
maybe add the build from source instructions here

Reply:
Yes. I had done this but not pushed it.